services-helpers.md

```diff
@@ -393,6 +394,13 @@ The `database_path` function returns the fully qualified path to the application
 
     $path = database_path();
 
+<a name="method-media-path"></a>
+#### `media_path()` {#collection-method}
+
+The `media_path` function returns the fully qualified path to the application media directory:
+
+    $path = media_path();
+
```

Comment:
Can you provide examples of input output? Also does this function support being passed an argument? I know some of the other function docs don't mention their support for arguments but it would be helpful if all of them that did mentioned it.

Reply:
done!